Makefile

```diff
@@ -38,8 +39,12 @@ manager: generate fmt vet
 
 # Build tool binary using GoReleaser in a local dev environment (in CI we just invoke GoReleaser directly)
 tool:
-	BUILD_METADATA=${BUILD_METADATA} SETUPTOOLS_IMG=${SETUPTOOLS_IMG} PULL_POLICY=${PULL_POLICY} REDSKYCTL_IMG=${REDSKYCTL_IMG} \
-	goreleaser release --snapshot --skip-sign --rm-dist
+	BUILD_METADATA=${BUILD_METADATA} \
+	SETUPTOOLS_IMG=${SETUPTOOLS_IMG} \
+	PULL_POLICY=${PULL_POLICY} \
+	REDSKYCTL_IMG=${REDSKYCTL_IMG} \
+	IMG=${IMG} \
+	goreleaser release --snapshot --skip-sign --rm-dist --debug
```

Comment:
Is adding `--debug` here intentional?

Reply:
I know we don't use this in CI, but in testing out the code signing, it definitely prints sensitive information.

Reply:
It shouldn't be there :)